docs/mkdocs.yml

```diff
@@ -5,6 +5,9 @@ repo_url: http://github.com/dssg/triage
 edit_uri: blob/master/docs/templates
 site_description: 'Documentation for Triage.'
 
+markdown_extensions:
+    - inline_graphviz
```

Comment:
Not sure why I'm getting `Config value: 'markdown_extensions'. Error: Failed loading extension "inline_graphviz".` when I ran `mkdocs serve`.

Reply:
Same issue here.

Not sure if [this applies](https://github.com/sprin/markdown-inline-graphviz/issues/6).

@thcrock Any reason you can think of that we'd see that discrepancy between our machines?

Reply:
The change suggested in that Github issue, changing the markdown extension reference to "mdx_inline_graphviz," gets docs to serve.

However, this is the error I see when I navigate to the architecture docs page (`/experiments/architecture/`):

![image](https://user-images.githubusercontent.com/530998/51991826-89019f00-2471-11e9-8f91-34fbc22a1dc9.png)

Reply:
I wonder if the difference in my machine is that I have graphviz itself installed (on the system!) while it's probably not in requirements. I'll look into it

Reply:
I don't see a possibility for pure python graphviz: achieving this inline may be impossible without also having the user install graphviz on their own. The `graphviz` python package generates DOT files using Python but depends on the dot executable to run. 

It may be easier to just bundle the DOT files, render these as images, and making it a needed step to rerender them to update the doc. Not as cool as inline, but may be necessary.

Reply:
Or....it looks like instead of graphviz I can do this with Mermaid: https://github.com/squidfunk/mkdocs-material/issues/693

Reply:
It actually looks way better with mermaid, even though there is a bit more work to get it started in a doc. Pushing changes soon